.github/workflows/publish_documentation.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish documentation
+
+# publish the documentation on every merge to develop branch
+on:
+  push:
+    branches:
+      - develop
+
+# we don't want to have concurrent jobs, and we don't want to cancel running jobs to avoid broken publications
+concurrency:
+  group: documentation
+  cancel-in-progress: false
+
+jobs:
+  publish_documentation:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      # We need a recent PlantUML version to generate UML diagrams. The version from Ubuntu is outdated, so we
+      # need to replace the JAR file with a release from PlantUML's GitHub page.
```

Comment:
Is that true for `ubuntu-22.04`?

Reply:
Is that newer than ubuntu-latest?

Reply:
In `ubuntu-22.04` the version is 1.2020.2 and in `ubuntu-latest` (20.04), it is 1.2018.13.

Reply:
I'll give it a try.

Reply:
Nice - this makes things much easier and with fewer dependencies.